pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,43 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return an ndarray of native Python datetime objects.
```

Comment:
I'd use something like "Return the data as native Python datetime objects".

The context helps, but the way is written makes me feel more like it's returning an empty or random array, not transforming data.